FreeRTOS/Test/VeriFast/list/generated/vListInsertEnd.c

```diff
@@ -0,0 +1,170 @@
+void vListInsertEnd( List_t * const pxList,
+                     ListItem_t * const pxNewListItem )
+    //@ requires List_t(pxList, ?len, ?idx, ?end, ?cells) &*& ListItem_t(pxNewListItem, ?val, _, _, _) &*& (mem(pxNewListItem, cells) == false);
+    //@ ensures List_t(pxList, len+1, idx, end, ?newcells) &*& (remove(pxNewListItem, newcells) == cells);
```

Comment:
Can we strengthen this postcondition so that we have `newcells == append(take(i, cells), append(cons(pxNewListItem, nil), drop(i, cells))` where `i == index_of(idx, cells)`?